src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,14 @@ public function getConfigTreeBuilder()
         $rootNode = $treeBuilder->root('web_profiler');
 
         $rootNode
+            ->beforeNormalization()
+                ->ifTrue(function ($v) { return array_key_exists('position', $v); })
```

Comment:
you can put the ``beforeNormalization`` in the ``position`` node directly, with a ``->always()`` condition